src/alchemlyb/convergence/convergence.py

```diff
@@ -94,7 +94,9 @@ def forward_backward_convergence(df_list, estimator="MBAR", num=10, **kwargs):
         for data in df_list:
             sample.append(data[: len(data) // num * i])
         sample = concat(sample)
-        result = estimator_fit(sample)
+        result = estimator_fit.fit(sample)
+        if estimator == "MBAR":
+            estimator_fit.initial_f_k = result.delta_f_.iloc[0, :]
```

Comment:
I didn't fully follow the code path but how is `delta_f` already initialized with the BAR results?

Reply:
So you did a MBAR calculation first `result = estimator_fit.fit(sample)`
Which populates the `result.delta_f_` with for the current MBAR results.
This MBAR results is then propagated back as initial guess for the next calculation.